### PR TITLE
OpenStack: Add new triage bot

### PR DESCRIPTION
**Pull request by stephenfin**
Depends on https://github.com/shiftstack/bugwatcher/pull/49

/hold

### CODE REVIEW

File: ci-operator/config/shiftstack/bugwatcher/shiftstack-bugwatcher-main.yaml [hunk 1]
```diff
@@ -38,6 +39,11 @@ tests:
   steps:
     test:
     - ref: shiftstack-bugwatcher-pretriage
+- as: bugwatcher-triage
+  cron: 0 8 * * *
```
Comment (mandre) [inline ci-operator/config/shiftstack/bugwatcher/shiftstack-bugwatcher-main.yaml:43]: How about we don't run this on the weekend, similar to the [bugwatcher-doctext job above](https://github.com/openshift/release/pull/38053/files#diff-9bd198042384d39bcafc3762f80f03d90910f77e7a701b4bdcd4d6ac39164464R33)?

Comment (stephenfin) [reply ci-operator/config/shiftstack/bugwatcher/shiftstack-bugwatcher-main.yaml:43]: Good point. Perhaps an arbitrary day? Wednesdays?

Comment (stephenfin) [reply ci-operator/config/shiftstack/bugwatcher/shiftstack-bugwatcher-main.yaml]: Chose Wednesday

**Comment by stephenfin** [comment]
/retest

This feature has now merged https://github.com/shiftstack/bugwatcher/pull/49

**Review by mandre — COMMENTED** [review]
You need to run "make jobs"

**Review by mandre — COMMENTED** [review]
/lgtm
/approve

**Comment by pierreprinetti** [comment]
ew that looks bad!

```
fatal error: concurrent map writes
```

**Comment by pierreprinetti** [comment]
This should fix the concurrent map access: https://github.com/shiftstack/bugwatcher/pull/59

**Comment by stephenfin** [comment]
/retest

@pierreprinetti resolved the issue in https://github.com/shiftstack/bugwatcher/pull/59

**Comment by stephenfin** [comment]
/pj-rehearse periodic-ci-shiftstack-bugwatcher-main-bugwatcher-triage

**Comment by mandre** [comment]
@stephenfin is this ready to merge?

**Comment by stephenfin** [comment]
> @stephenfin is this ready to merge?

Sure is!

**Review by mandre — COMMENTED** [review]
/lgtm

**Review by mandre — COMMENTED** [review]
/lgtm

**Pull request merged**